src/models/acopf/acopf_admm_update_z_ka.jl

```diff
@@ -0,0 +1,28 @@
+@kernel function update_zv_kernel_ka(n::Int, u, v, z, l, rho, lz, beta)
+    I = @index(Group, Linear)
+    J = @index(Local, Linear)
+    tx = J + (@groupsize()[1] * (I - 1))
+
+    if tx <= n
+        @inbounds begin
```

Comment:
Here we are using `@inbounds`, but not in the previous kernels. Would it be possible to use `@inbounds` systematically?